internal/core/ui/src/main/res/values/accessibility-strings.xml

```diff
@@ -0,0 +1,13 @@
+<resources>
```

Comment:
We might need a German translation for the talk back to handle this

Reply:
all these copies needs to be verified by translators, let's see how we can make accessibility copies part of the translation process 👍